invenio_accounts/models.py

```diff
@@ -78,6 +80,10 @@ class User(db.Model, UserMixin):
     roles = db.relationship('Role', secondary=userrole,
                             backref=db.backref('users', lazy='dynamic'))
 
+    active_sessions = db.relationship('SessionActivity', backref='user')
```

Comment:
it's better to define it on `SessionActivity` model  as `user = db.relationship(User, backref='active_sessions')` (please note that you can use the class directly not its string name).